engines/groovie/detection.cpp

```diff
@@ -180,7 +191,21 @@ static const GroovieGameDescription gameDescriptions[] = {
 			Common::EN_ANY, Common::kPlatformMacintosh, ADGF_UNSTABLE,
 			GUIO4(GUIO_MIDIADLIB, GUIO_MIDIMT32, GUIO_MIDIGM, GUIO_NOASPECT)
 		},
-		kGroovieV2, 1
+		kGroovieT11H, 1
+	},
+
+	// The 11th Hour Windows English (Available on Steam)
+	{
+		{
+			"11h", "Digital",
+			{
+				{ "disk.1", 0, "4c1d0549f544f052fba2b7a9aebd1077", 220},
```

Comment:
For this, use `AD_ENTRY1s()` macro, or better `AD_ENTRY2s()` for two files as mentioned above.